polaris-react/src/components/Button/Button.scss

```diff
@@ -893,6 +893,7 @@
       margin: 0;
       padding: 0;
       min-height: var(--p-font-line-height-2);
+      min-width: var(--p-font-line-height-2);
```

Comment:
Is it ok that I'm using a line height token here or should I just hard code `20px` for both of these values? It should be dependent on the line-height because we want to align with it